eng/templates/Azure.ServiceTemplate.Template/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
+  -->
```

Comment:
Can we avoid having this type in the template?

Reply:
Do you mean the comment?

Reply:
Sorry, I meant to say this file.

Reply:
We need this file. To build Each SDK package, we use those files to find all the dependencies.

Reply:
What breaks if we remove it?

Reply:
Yes. After remove it, when we build the SDK client library, it will not be able to find out the build dependencies and properties. 
<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
Above statement in the file is to find out all the build props which is defined in sdk root directory

Reply:
confirmed with Wes, the file does not need. remove it.